BuildLoader.py

```diff
@@ -1114,8 +1114,9 @@ def pre_build(self):
             if getattr(self._board, "GetKeyHashList", None):
                 key_hash_list = self._board.GetKeyHashList ()
 
+        svn = 0

```

Comment:
Does this needs to be configured from Board config file ?

Reply:
Thanks. Yes we could get it from boardconfig files as new config param for KEYH container svn.